Sources/StreamChat/Audio/AudioSessionConfiguring.swift

```diff
@@ -68,15 +68,17 @@ open class StreamAudioSessionConfigurator: AudioSessionConfiguring {
     /// Calling this method should activate the provided `AVAudioSession` for recording and playback.
     ///
     /// - Note: This method is using the `.playAndRecord` category with the `.spokenAudio` mode.
-    /// The preferredInput will be set to `.buildInMic` and overrideOutputAudioPort to `.speaker`.
     open func activateRecordingSession() throws {
         try audioSession.setCategory(
             .playAndRecord,
             mode: .spokenAudio,
             policy: .default,
-            options: []
+            options: [
+                // It is deprecated, but for now we need to use it,
+                // since the newer ones are not available in Xcode 15.
+                .allowBluetooth
+            ]
         )
-        try setUpPreferredInput(.builtInMic)
```

Comment:
That's just a preference and not really setting the mic to the builtIn (at least not always). You are right though, better not to have any preference here.